examples/valid/plots_list.dvc.yaml

```diff
@@ -20,8 +20,20 @@ plots:
       y:
         evaluation/train/plots/roc.json: tpr
         evaluation/test/plots/roc.json: tpr
+  - confusion: # x dict
+      x:
+        dir/actual.csv: actual
+      y:
+        dir/preds.csv: predicted
+      template: confusion - evaluation/importance.png # Plot an image as str.
```

Comment:
Is this legal? Or, is it just an example?

Reply:
Looks like I just messed up editing the file.